pkgs/by-name/ht/htb-toolkit/package.nix

```
@@ -0,0 +1,85 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, openssl
+, stdenv
+, darwin
+, coreutils
+, noto-fonts-color-emoji
+, gnome
+, libsecret
+, bash
+, openvpn
+, nerdfonts
+, gzip
+, killall
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "htb-toolkit";
+  version = "unstable-2024-01-17";
+
+  src = fetchFromGitHub {
+    owner = "D3vil0p3r";
+    repo = "htb-toolkit";
+    rev = "54e11774ea8746ea540548082d3b25c22306b4fc";
+    hash = "sha256-QYUqdqFV9Qn+VbJTnz5hx5I0XV1nrzCoCKtRS7jBLsE=";
+  };
+
+  cargoHash = "sha256-XDE6A6EIAUbuzt8Zb/ROfDAPp0ZyN0WQ4D1gWHjRVhg=";
+
+  patchPhase = ''
+    runHook prePatch
+    patch -p1 < ${./main.patch}
+    patch -p1 < ${./manage.patch}
+    patch -p1 < ${./play.patch}
+    patch -p1 < ${./types.patch}
+    patch -p1 < ${./utils.patch}
+    runHook postPatch
+  '';
```

Simply listing the patches should be sufficient.

```suggestion
  patches = [
    ./main.patch
    ./manage.patch
    ./play.patch
    ./types.patch
    ./utils.patch
  ];
```

If the patches are only relevant for Nixpkgs then a comment that mention this would help in the future.